src/main.rs

```diff
@@ -576,7 +575,7 @@ impl fmt::Display for FDTarget {
                     .collect::<Vec<String>>()
                     .join(",");
 
-                format!("pipe {direction} {joined_endpoints}")
+                format!("pipe {joined_endpoints}")
```

Comment:
Let's add an arrow here. The direction is now redundant with the mode, but without the arrow, it's confusing to know what `pipe [12]` means.
```suggestion
                format!("pipe -> {joined_endpoints}")
```